e2e/scenario/images.go

```diff
@@ -10,3 +10,9 @@ var DefaultImageVersionIDs = map[string]string{
 	"marinerv2-arm64":    "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/CBLMarinerV2Gen2Arm64/versions/1.1699146990.16638",
 	"azurelinuxv2-arm64": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/AzureLinuxV2Gen2Arm64/versions/1.1699146999.31863",
 }
+
+var DefaultGPUSeriesSKU = map[string]string{
```

Comment:
nit: `DefaultGPUSeriesVMSizes`?

Reply:
sounds good, do you think the file name should be changed? wasn't sure what a better name would be

Reply:
I wonder if a separate file for this would be better - like gpuvmsizes.go or it can also be moved to e2e/const.go

Reply:
maybe we could just do `vm.go` or similar